lib/node_modules/@stdlib/constants/float32/sqrt-three/README.md

```diff
@@ -0,0 +1,135 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# FLOAT32_SQRT3
+
+> [Square root][@stdlib/math/base/special/sqrt] of `3` as a single-precision floating-point number.
```

Comment:
```suggestion
> [Square root][@stdlib/math/base/special/sqrtf] of `3` as a single-precision floating-point number.
```

Should link to the single-precision `sqrtf` package, not the double-precision package.

Reply:
That applies here and below.